Tech Insights documentation (#881)
RoadieHQ/marketing-site: content/docs/tech-insights/scorecards/index.md

---
title: Checks
publishedDate: '2022-11-15'
description: Managing Scorecards.
---

# Managing Scorecards

Scorecards are collections of multiple Checks, which may come from different Data Sources.

> Overview and manage your Scorecards via Tech Insights → Scorecards page. In this page you can also add Scorecards.

Reviewer: I'm not sure 'Overview' can be used as a verb like this.
Maybe 'You can see the overview and manage your...'?

> Overview and manage your Scorecards via Tech Insights → Scorecards page. In this page you can also add Scorecards.

Reviewer: ```suggestion
Overview and manage your Scorecards via Tech Insights → Scorecards page. In this page you can also create new Scorecards.
```

Reviewer: (the sentence structure is duplicated in the paragraph below)

You can also add Scorecards to the Catalog page so they’re visible to your team.

![Scorecard overview](./scorecard-overview.png)

Clicking the title of the scorecard will lead to scorecard results in Backstage catalog components.

![Scorecard catalog](./scorecard-result.png)

### Adding a Scorecard

> Once you click in the “Add Scorecard” button, you’ll be prompted to fill in the details about the Scorecard, including the Checks you want to include and to which entities to apply it to, as shown below:

Reviewer: ```suggestion
Once you click the “Add Scorecard” button, you’ll be prompted to fill in the details about the Scorecard, including the Checks you want to include and to which entities to apply it to, as shown below:
```

![Create Scorecard](./create-scorecard.png)

> 1 - General information about scorecard (Name and description).

Reviewer: ```suggestion
1 - General information about scorecard (Name and Description).
```

This is capped in other sections

> 2 - Checks you wish to include in the scorecard. These checks can be created before and added by clicking ‘Add’ button in this section, which will lead to opening this dialog, in which you select all the checks you want

Reviewer: This runs a bit long
```suggestion
2 - Checks you wish to include in the scorecard. These checks can be created before and added by clicking ‘Add’ button in this section, which will lead to opening this dialog. Within the dialog you can select all the checks you want
```

Reviewer: no wait it continues after the image

![Add check](./add-check.png)

> or added later through ‘Edit’ functionality.

Reviewer: ```suggestion
or you can add additional Checks later through ‘Edit’ functionality.
```

![Edit Scorecard](./edit-scorecard.png)

3 - Select which kind and type of components this scorecard applies to.

4 - Owner which can be either a user or a group.

Once you are happy with the created scorecard, save it and you should be able to see it in overview screen.

### Adding a Scorecard to the Catalog

> In order to show a Scorecard in the Catalog Entity page, as shown below, you’ll have to define it from the Tech Insights → Scorecards page.

Reviewer: ```suggestion
In order to show a Scorecard in the Catalog Entity page, you’ll have to define it from the Tech Insights → Scorecards page.
```

It's not shown. Missing screenshot?

Author: Yeah, wrong one. Thanks

![Scorecard Overview](./scorecard-overview.png)

Once you’re Tech Insights → Scorecards, find a Scorecard you wish to add. From kebab menu select ‘Add to Catalog’

![Add Scorecard to catalog](./add-to-catalog-scorecard.png)

> The select option which best applies to your use case.

Reviewer: ```suggestion
Then select option which best applies to your use case.
```
